src/core/PRINCIPLES.md

```diff
@@ -0,0 +1,47 @@
+## Common dictionary
+Plugin - a piece of software expressing specific subject of the business unit within the code.
+
+Platform - a set of plugins required to run all other Kibana plugins.
```

Comment:
calling core's elasticsearch a "plugin" can be a bit confusing.

This from line 24 could make it sound like you need to declare an dependency on core's elasticsearch too.
> Each plugin should declare dependencies on the other plugins explicitly

Maybe we could call these services instead of plugins?

Reply:
`services` introduces a new term. how about `functionality required to run all the Kibana plugins.`?